Scripts/DCS-BIOS/lib/modules/aircraft_modules/FC3.lua

```diff
@@ -0,0 +1,517 @@
+module("FC3", package.seeall)
+
+local Module = require("Module")
+
+--- @class FC3: Module
+local FC3 = Module:new("FC3", 0x6000, BIOS.FLAMING_CLIFFS_AIRCRAFT)
+
+local altitude = " -- "
+local altitude_ground_level = " ---- "
+local _altitude_sea_level = " ---- "
+local radar_altitude = 0
+
+local vertical_velocity = " -- "
+local indicated_airspeed = " -- "
+local true_air_speed = " -- "
+local mach_number = " -- "
+local vvi_bar = 8
+
+local angle_of_attack = " -- "
+
+local fuel_bar = 0
+local fuel_percentage = " -- "
+
+local gload = 0
+local gload_bar = 0
+
+local adi_bank = 0.0
+local adi_pitch = 0.0
+local adi_yaw = 0.0
+
+local glide_deviation
+local side_deviation
+local slip_ball_position
+
+local chaffs
+local flares
+
+local gear_status
+local left_rpm = "----"
+local right_rpm = "----"
+local left_temp = "----"
+local right_temp = "----"
+local left_hydraulic = "---"
+local right_hydraulic = "---"
+local left_fuel_consumption = "----"
+local right_fuel_consumption = "----"
+
+local function GetPlaneName()
+	local selfdata = LoGetSelfData()
+	if selfdata == nil then
+		return "XXX"
+	end
+	return selfdata.Name
+end
+
+local function LoGetFuelAll()
+	local engine = LoGetEngineInfo()
+	if engine == nil then
+		return
+	end
+	return engine.fuel_internal + engine.fuel_external
+end
+
+local function BarFuel(fuel, plane)
+	local maxFuel = 1000
+
+	if plane == "A-10A" then
+		maxFuel = 5029
+	elseif plane == "F-15C" then
+		maxFuel = 6103
+	elseif plane == "MiG-29A" or plane == "MiG-29G" then
+		maxFuel = 3380
+	elseif plane == "MiG-29S" then
+		maxFuel = 3500
+	elseif plane == "Su-25" then
+		maxFuel = 2835
+	elseif plane == "Su-25T" then
+		maxFuel = 3790
+	elseif plane == "Su-27" or plane == "Su-33" or plane == "J-11A" then
+		maxFuel = 9400
+	end
+
+	local barf = math.ceil((fuel / maxFuel) * 16)
+	if barf > 16 then
+		barf = 16
+	end
+	if barf == nil then
+		return
+	end
+	return barf
+end
+
+local function BarGLoad(gload)
+	local barg = math.floor((gload / 11) * 16)
+
+	if barg < 0 then
+		barg = 0
+	elseif barg > 16 then
+		barg = 16
+	end
+	if barg == nil then
+		return
+	end
+	return barg
+end
+
+local function BarVVI(vvi, plane)
+	local divide = 150
+
+	if plane == "A-10A" or plane == "F-15C" then
+		divide = 30
+	end
+	local bar = (vvi / divide) * 8
+	if bar < -8 then
+		bar = -8
+	elseif bar < 0 then
+		bar = math.floor(bar)
+	elseif bar > 8 then
+		bar = 8
+	elseif bar > 0 then
+		bar = math.ceil(bar)
+	end
+	if bar == nil then
+		return
+	end
+	return bar + 8
+end
+
+local function allowMach(plane)
+	if plane == "A-10A" then
+		return false
+	elseif plane == "Su-25T" then
+		return false
+	else
+		return true
+	end
```

Comment:
this can be condensed to
```suggestion
	 return plane ~= "A-10A" and plane ~= "Su-25T
```